src/ansys/aedt/core/visualization/advanced/misc.py

```diff
@@ -331,6 +331,14 @@ def get_point(ll, start, length):
             elif line_type in ["CPYRA", "CPYRAM", "CPYRA*", "CPYRAM*"]:
                 num_points = 5
                 obj_type = "Solids"
+            elif line_type in ["CHEXA", "CHEXA*"]:
+                num_points = 8
+                obj_type = "Solids"
+            elif line_type in ["PSHELL", "PSHELL*"]:
+                num_points = 1
+                obj_type = "Shells"
+            elif line_type in ["MPC", "MPC*"]:
+                num_points = 16
```

Comment:
Shouldn't you affect `obj_type` here also ?

Reply:
nope because I don't that info for MCP